schema/backwards_compatibility_test.go

```diff
@@ -100,8 +100,7 @@ func TestBackwardsCompatibilityImageIndex(t *testing.T) {
          "digest": "sha256:fb2fc0707b86dafa9959fe3d29e66af8787aee4d9a23581714be65db4265ad8a",
          "platform": {
             "architecture": "arm64",
-            "os": "linux",
-            "variant": "armv8"
```

Comment:
A question then - is the OCI specification subordinate to go with respect to the value of attributes here?  Would this not make the implementation of this specification sensitive to changes in go?
I thought that the metadata in this specification was to define the compatibility of the images - in which case we ought to be complete in our definition here.  The full definition in this instance would be { "arch": "arm64", "variant": "8" }?